packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -39,7 +41,7 @@ type snackbarData = {
  *
  * In the future, this may change (e.g. allowing non-indefinite `duration`)
  */
-type modifyToastOptions = snackbarData
+export type modifyToastOptions = snackbarData
 
 export type SnackbarType = Omit<ToastType, 'show' | 'update'> & {
```

Comment:
Think this can be deleted as it was used just for defining the global `snackbar` type.